dotnet/content/install.cmd

```diff
@@ -1,7 +1,17 @@
+
+REM Entrypoint: https://github.com/projectkudu/kudu/blob/master/Kudu.Core/SiteExtensions/SiteExtensionManager.cs#L548
+
 @echo off
 
 echo Starting install
 
 POWERSHELL .\install.ps1
 
+IF EXIST ".\installation-failure.txt" (
+  echo Install failure
+  exit /B 1
+)
```

Comment:
Not a blocker for this PR, but would it possible later to return an error code from `install.ps1` instead of writing to a file?

Reply:
Noted for future work.

Reply:
No longer doing this, the presence of applicationHost.xdt indicates success.